WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -209,6 +216,10 @@ public static PSBT ParsePsbt(string json, Network network)
 		if (JsonHelpers.TryParseJToken(json, out JToken? token))
 		{
 			var psbtString = token["psbt"]?.ToString()?.Trim() ?? null;
+			if (string.IsNullOrWhiteSpace(psbtString))
+			{
+				throw new ArgumentNullException(nameof(psbtString), "Failed to parse PSBT, parameter was null.");
```

Comment:
I wouldn't say the parse failed:

```suggestion
				throw new ArgumentNullException(nameof(psbtString), "PSBT parameter was null.");
```